enterprise_access/apps/subsidy_access_policy/models.py

```diff
@@ -809,13 +823,18 @@ def redeem(self, lms_user_id, content_key, all_transactions, metadata=None, **kw
                 subsidy_access_policy_uuid=str(self.uuid),
                 historical_redemptions_uuids=self._redemptions_for_idempotency_key(all_transactions),
             )
+            # If this policy has late redemptions currently enabled, tell that to the subsidy service.
+            metadata_for_tx = metadata
+            if self.is_late_redemption_allowed:
+                metadata_for_tx = metadata.copy() if metadata else {}
+                metadata_for_tx['late_enrollment'] = True
```

Comment:
I thought the key was `allow_late_enrollment`?
https://github.com/openedx/enterprise-subsidy/pull/219/files